packages/office-ui-fabric-react/src/components/Spinner/Spinner.Props.ts

```diff
@@ -20,6 +20,30 @@ export interface ISpinnerProps extends React.Props<Spinner> {
 }
 
 export enum SpinnerType {
-  normal = 0,
-  large = 1
+
+  /**
+   * 12px Spinner diameter
+   */
+  xSmall = 0,
+
+  /**
+   * 16px Spinner diameter
+   */
+  small = 1,
+
+  /**
+   * 20px Spinner diameter
+   */
+  medium = 2,
+
+  /**
+   * @deprecated
+   * Deprecated and will be removed at >= 2.0.0. Use medium instead.
+   */
+  normal = 2,
```

Comment:
Might need @dzearing to weigh in here, but changing normal from 0 to 2 and large from 1 to 3 could cause some problems. I.E. the numbers should never change (unless deprecated), and the numbers don't have to be in size order.

normal 0
large 1
xsmall 2
small 3
medium 4

This maintains backwards compat.